### DISCUSSION TITLE
Is it possible to make carousel with more than 1 slide at a time with outer controls?

### DISCUSSION THREAD

**Discussion by tesicg** [bootstrap-vue/bootstrap-vue#6327] [Q&A] [answered]
It should look as following:

https://i.postimg.cc/Z5H7gtjt/Screenshot-2021-01-18-090028.jpg

**Comment by Hiws** [comment, marked as answer]
You can place multiple images into one slide, by using the `img` slot.

Read more here: <https://github.com/bootstrap-vue/bootstrap-vue/issues/4198#issuecomment-538659518>
